Source/WPE/Source/Input/LibinputServer.cpp

```diff
@@ -108,12 +113,33 @@ void LibinputServer::setPointerBounds(uint32_t width, uint32_t height)
     m_pointerBounds = { width, height };
 }
 
+void LibinputServer::configureHandleTouchEvents(bool enable, uint32_t touchWidth, uint32_t touchHeight)
+{
+    m_handleTouchEvents = enable;
+    if (enable)
+        m_touchSize = { touchWidth, touchHeight };
```

Comment:
m_touchSize implies more that it's containing the size of the touch, instead it's really the size bounds in which the touch events are generated. Ergo, this variable should be named m_touchBounds.

Ideally it would be folded together with m_pointerBounds into a m_inputBounds variable.